Source/ui_qt/Qt_ui/settingsdialog.ui

```diff
@@ -205,28 +205,60 @@
         </widget>
        </item>
        <item>
-        <widget class="QCheckBox" name="checkBox_enable_vulkan">
-         <property name="text">
-          <string>Enable Vulkan GS Handler</string>
-         </property>
-        </widget>
-       </item>
-       <item>
-        <widget class="QLabel" name="label_5">
-         <property name="text">
-          <string>Warning: The Vulkan GS handler requires the VK_EXT_fragment_shader_interlock extension to work properly.
+        <widget class="QWidget" name="gs_option_widget" native="true">
+         <layout class="QGridLayout" name="gridLayout_2">
+          <property name="leftMargin">
+           <number>0</number>
+          </property>
+          <property name="topMargin">
+           <number>0</number>
+          </property>
+          <property name="rightMargin">
+           <number>0</number>
+          </property>
+          <property name="bottomMargin">
+           <number>0</number>
+          </property>
+          <item row="0" column="0">
+           <widget class="QLabel" name="label_6">
+            <property name="text">
+             <string>GS Render:</string>
+            </property>
+           </widget>
+          </item>
+          <item row="1" column="0">
+           <widget class="QComboBox" name="comboBox_gs_selection">
+            <item>
+             <property name="text">
```

Comment:
Might be interesting to consider to populate that from the code and reuse the same logic that is used in the status bar code. We might only have 2 GS handlers for a long time, but let's make this easy to add a 3rd one if we need.